advocacy_docs/edb-postgres-ai/migration-etl/data-migration-service/getting_started/config_reader.mdx

```diff
@@ -85,64 +89,70 @@ Consider the following ID guidelines:
 - The ID must be unique. The source instances cannot have the same ID.
 
-### RW_SERVICE_HOST
+### `RW_SERVICE_HOST`
 
 Specifies the URL of the service that will host the migration. `transporter-rw-service` is always https://transporter-rw-service.biganimal.com.
 
-### TLS_PRIVATE_KEY_PATH
+### `TLS_PRIVATE_KEY_PATH`
 
 Directory path to the `client-key.pem` private key you downloaded from the EDB Postgres AI Console. 
-The Reader's HTTP client uses it to perform mTLS authentication with the `transporter-rw-service`.
 
-### TLS_CERTIFICATE_PATH
+The HTTP client of the EDB DMS Reader uses it to perform mTLS authentication with the `transporter-rw-service`.
+
+### `TLS_CERTIFICATE_PATH`
 
 Directory path to the X509 `client-cert.pem` certificate you downloaded from the EDB Postgres AI Console.
-The Reader's HTTP client uses it to perform mTLS authentication with the `transporter-rw-service`.
 
-### TLS_CA_PATH
+The HTTP client of the EDB DMS Reader uses it to perform mTLS authentication with the `transporter-rw-service`.
+
+### `TLS_CA_PATH`
 
-Directory path to the `int.cert` Certificate Authority you downloaded from the EDB Postgres AI Console. 
-It signs the certificate configured in TLS_CERTIFICATE_PATH.
+Directory path to the `int.cert` Certificate Authority you downloaded from the EDB Postgres AI Console.
 
-### APICURIOREQUEST_CLIENT_KEYSTORE_LOCATION
+It signs the certificate configured in `TLS_CERTIFICATE_PATH`.
+
+### `APICURIOREQUEST_CLIENT_KEYSTORE_LOCATION`
 
 Directory path to the `client-keystore.p12` keystore location file you downloaded from the EDB Postgres AI Console.
-It is created from the private key and certifiate configured in TLS_PRIVATE_KEY_PATH and TLS_CERTIFICATE_PATH.
+It is created from the private key and certificate configured in `TLS_PRIVATE_KEY_PATH` and `TLS_CERTIFICATE_PATH`.
+
 The Apicurio client uses it to perform mTLS authentication with the `transporter-rw-service`.
 
-### APICURIOREQUEST_TRUSTSTORE_LOCATION
-Created from the Certificate Authority configured in TLS_CA_PATH
-Apicurio client use it to mTLS with transporter-rw-service
+### `APICURIOREQUEST_TRUSTSTORE_LOCATION`
+
+Created from the Certificate Authority configured in `TLS_CA_PATH`.
+
+The Apicurio client uses it to perform mTLS authentication with the `transporter-rw-service`.
+
+### `DBZ_DATABASES`
+
+This is a list of source database information you require for the EDB DMS Reader be able to read the correct source database information for the migration. 
+
+You can configure the EDB DMS Reader to migrate multiple databases. The `DBZ_DATABASES_0__TYPE` section delimits the information for the first database. You can use `DBZ_DATABASES_1__TYPE` to provide data for a second database. Add more sections to the EDB DMS Reader (`DBZ_DATABASES_2__TYPE`, `DBZ_DATABASES_3__TYPE`) by increasing the index manully.
```

Comment:
I didn't notice before, but I'm not sure why we use double underscores for these. @jperozoa I couldn't figure it out at a glance, do you know?

Reply:
maybe @zahid75 knows?

Reply:
I didn't try (as a matter of fact I was not aware of it) it earlier, will confirm now

Reply:
I confirmed, one can use the same reader instance to point to multiple source databases (by using _DATABASES_N__ pattern), and each database will appear as a separate source e.g. I defined two source PG sources and each appeared as a separate source on Sources page e.g.  "ora2pg (Postgres) zi-p2e-aug29-2" and "soe (Postgres) zi-p2e-aug29-2".